nuxt.config.ts

```diff
@@ -9,7 +9,12 @@ export default defineNuxtConfig({
             fileURLToPath(new URL('./src/config.ts', import.meta.url))
           ]
         }
-      }
-    }
+      },
+      exclude: [
+        fileURLToPath(new URL('./docs', import.meta.url))
+      ]
```

Comment:
Exclude type checking the local `./docs/` directory; doesn't seem to resolve everything correctly and doesn't impact the actual module release.